workflows/kfdrc-germline-snv-wf.cwl

```diff
@@ -423,7 +423,7 @@ steps:
       run_gatk: boolean_to_boolean_gvcf/out_bool
       input_bam:
         source: [samtools_view/output, input_reads]
-        pickValue: first_non_null
+        valueFrom: '$(self[0] ? self[0] : self[1])'
```

Comment:
Now that I know of it, does the elvis format work here?

Reply:
There is no elvis the version of javascript that CWL uses